premium/usage.go

```diff
@@ -401,11 +481,19 @@ func (u *BatchUpdater) backgroundUpdater() {
 			case <-u.done:
```

Comment:
Do we not need the `roundDown` logic in the `case <-u.flushDuration.C:` case as well?